deny.toml

```diff
@@ -15,7 +15,6 @@ allow = [
     "BSD-2-Clause",
     "BSD-3-Clause",
     "MIT",
-    "Unicode-DFS-2016",
```

Comment:
CI is still failing https://circleci.com/gh/influxdata/rskafka/240564

Maybe this needs to be changed to "Unicode-3.0" rather than deleted

Reply:
fixed